rollup-cli/tools/bot.js

```diff
@@ -1,58 +1,67 @@
 const { Wallet } = require("../src/wallet");
-const { argv } = require('yargs'); // eslint-disable-line
+const { argv } = require("yargs").alias("t", "tokens").alias("mn", "mnemonic").alias("p", "path").alias("e", "ether")
+    .alias("w", "wallets").alias("d", "deposits").alias("s", "sends");
 const fs = require("fs");
-const { depositTx, sendTx } = require("../src/cli-utils");
-const configjson = "../src/resources/config.json";
+const configjsonBot = "../tools/resourcesBot/configBot.json";
+const ethers = require("ethers");
+const {createWallets} = require("../tools/createWallets");
```

Comment:
blank spaces before/after brackets:
`{ createWallet }`
Applies to this line and further lines